File: classes/Video.py
import classes.Frame as Frame
class Video:
    def __init__(self, frames, width, height):
        self.frames = frames
        self.width = width
        self.height = height
    def add_frame(self, frame):
        self.frames.append(frame)
    def __repr__(self):
        return f"Video({len(self.frames)} frames, {self.width}x{self.height})\n"

File: Csharp/Classes/Video.cs
using Classes.Frames;

namespace Classes.Videos
{
    public class Video
    {
        public List<Frame> frames { get; set; } = new();
        public int width { get; set; }
        public int height { get; set; }

        // Construtor padrao para desserializacao
        public Video() { }

        public Video(List<Frame> frames, int height, int width)
        {
            this.height = height;
            this.width = width;
            this.frames = frames;
        }
    }
}
